### PR TITLE
Fix Pylint ingestion/bulksink and ingestion/stage

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/src/metadata/ingestion/bulksink/migrate.py [hunk 1]
```diff
@@ -1,707 +0,0 @@
-#  Copyright 2021 Collate
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/bulksink/migrate.py:1]: Only used for migration from 0.9 to 0.10. Deprecating it as the implementation won't even work with current entities

File: ingestion/src/metadata/ingestion/stage/migrate.py [hunk 2]
```diff
@@ -1,127 +0,0 @@
-#  Copyright 2021 Collate
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/stage/migrate.py:1]: Only used for migration from 0.9 to 0.10. Deprecating it as the implementation won't even work with current entities

File: ingestion/src/metadata/ingestion/stage/file.py [hunk 3]
```diff
@@ -1,62 +0,0 @@
-#  Copyright 2021 Collate
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/stage/file.py:1]: not being used in any bulksink

**Review by ulixius9 — APPROVED** [review]

File: ingestion/tests/integration/stage/stage_test.py [hunk 4]
```diff
@@ -1,48 +0,0 @@
...
-        "type": "SampleData",
-        "sampleDataFolder": "ingestion/examples/sample_data"
-      }
-    },
-    "sourceConfig": {}
-  },
-  "stage": {
-    "type": "file",
-    "config": {
-      "filename": "/tmp/stage_test"
```
Comment (pmbrull) [inline ingestion/tests/integration/stage/stage_test.py:22]: this test validates a unused stage

**Pull request merged**